spaces/yeld/skin.scss

```
@@ -0,0 +1,10 @@
+.name {
```

Class name should be the same as the space key
```suggestion
.yeld {
```
Also the file should be in the repo root /skins/yeld.scss